src/components/text_input/text_input.xml

```diff
@@ -1,5 +1,10 @@
 <templates>
   <div t-name="o-spreadsheet-TextInput" class="w-100">
+    <!--
```

Comment:
nitpick: not sure but I'd maybe put the comment in the `.ts`. The problem is that comments in the xml will end up in the DOM.